honu_test.go

```diff
@@ -161,6 +164,20 @@ func TestLevelDBInteractions(t *testing.T) {
 		require.NoError(t, iter.Error())
 		iter.Release()
 	}
+
+	// Test iteration over all the namespaces
+	// FIXME: This is skipping the undeleted values
+	engine, ok := db.Engine().(*leveldb.LevelDBEngine)
+	require.True(t, ok)
```

Comment:
good type assertion!

Reply:
If you add a `totalKeys++` to line 110 after the undelete you'll see the count mismatch.